src/cli/commands/lint.ts

```diff
@@ -41,23 +45,28 @@ linting ./openapi.yaml
       char: 'o',
       description: 'output to a file instead of stdout',
     }),
+    max: flagHelpers.integer({
+      description: '[default: all] maximum results to show',
+    }),
+    // @deprecated in 2.2, remove in 3.0
     maxResults: flagHelpers.integer({
```

Comment:
If we decide it to drop it eventually, we should show a deprecation notice now.

Reply:
When we have more usage I will put more effort into this, but for now a changelog will suffice.